src/main/drivers/light_led.c

```diff
@@ -88,3 +92,35 @@ void ledSet(int led, bool on)
     const bool inverted = (1 << (led)) & ledInversion;
     IOWrite(leds[led], on ? inverted : !inverted);
 }
+
+#ifdef USE_SENSOR_HEATING
```

Comment:
I think this should go into a separate file - `light_led.c` seems to be the wrong place for a heating driver. (Also, if it's a separate file, it should probably only be included in `target.mk` files for targets that use it (for now).